setup.py

```diff
@@ -1,18 +1,22 @@
 #!/usr/bin/env python3
 
+"""Setup for python module."""
+
 import os
 import sys
 from setuptools import setup
 from pathlib import Path
-from setuptools.command.test import test as TestCommand
+from setuptools.command.test import test as TestCommand  # noqa: N812
```

Comment:
Any specific reason for ignoring the N812 code here?

Reply:
otherwise it pytest states `N812 lowercase 'test' imported as non lowercase 'TestCommand'`

Reply:
so why not change the `TestCommand` to something like `test_command` and use it wherever necessary?

Reply:
fixed